beefy-primitives/tests/light_client/validator_set.rs

```diff
@@ -29,7 +29,7 @@ pub enum Signature {
 	Invalid,
 }
 
-impl Signature {
+impl beefy_primitives::crypto::Signature {
```

Comment:
Hmm, okay, I see the issue now. We can't use custom `Public/Signature` key types any more, because we are bound to concrete types.
This is a bit backward change imho and I feel it will give us a hiccup in a near future.